07-topology-and-geocoding.Rmd

```diff
@@ -20,36 +20,76 @@ Frequently, we need spatial data to behave and relate and in specific and predic
 
 ### Key Terms {-}
 
-Vertex, Node, Pseudonode, Dangle, Planar Topology, Non-Planar Topology, Geocoding, Adjacency, Overlap, Connect, Inside, Reverse Geocoding
+Vertex, Node, Pseudonode, Dangle, Planar Topology, Non-Planar Topology, Geocoding, Adjacency, Overlap, Connect, Inside, Reverse Geocoding, small change
```

Comment:
Not sure if last keyword ("small change") is meant to be included of if is a commit message that was added to the wrong place.

Reply:
Odd, not sure how that ended up in there. Ditto for comment below.